tests/unit/test_exceptions.py

```diff
@@ -21,17 +25,74 @@ def test_str(self):
         assert str(PRAWException("foo")) == "foo"
 
 
-class TestAPIException:
-    def test_inheritance(self):
-        assert isinstance(APIException(None, None, None), PRAWException)
+class TestRedditErrorItem:
+    def test_equality(self):
+        resp = ["BAD_SOMETHING", "invalid something", "some_field"]
+        error = RedditErrorItem(*resp)
+        error2 = RedditErrorItem(*resp)
+        assert error == error2
+        assert error != 0
+
+    def test_property(self):
+        error = RedditErrorItem(
+            *["BAD_SOMETHING", "invalid something", "some_field"]
```

Comment:
What's the purpose of *[...]?

Reply:
RedditErrorItem has three arguments. If I wanted to, I could remove the list.